src/sentry/utils/snuba.py

```diff
@@ -791,29 +791,37 @@ def raw_snql_query(
     referrer: Optional[str] = None,
     use_cache: bool = False,
 ) -> Mapping[str, Any]:
+    """
+    Alias for `bulk_snuba_queries`, kept for backwards compatibility.
+    """
     # XXX (evanh): This function does none of the extra processing that the
     # other functions do here. It does not add any automatic conditions, format
     # results, nothing. Use at your own risk.
-    metrics.incr("snql.sdk.api", tags={"referrer": referrer or "unknown"})
-    if "consistent" in OVERRIDE_OPTIONS:
-        request.flags.consistent = OVERRIDE_OPTIONS["consistent"]
+    return bulk_snuba_queries([request], referrer, use_cache)[0]
 
-    if referrer:
-        request.tenant_ids = request.tenant_ids or dict()
-        request.tenant_ids["referrer"] = referrer
 
-    params: SnubaQueryBody = (request, lambda x: x, lambda x: x)
-    return _apply_cache_and_build_results([params], referrer=referrer, use_cache=use_cache)[0]
+def bulk_snql_query(
+    requests: List[Request],
+    referrer: Optional[str] = None,
+    use_cache: bool = False,
+) -> ResultSet:
+    """
+    Alias for `bulk_snuba_queries`, kept for backwards compatibility.
+    """
+    return bulk_snuba_queries(requests, referrer, use_cache)
 
 
-def bulk_snql_query(
+def bulk_snuba_queries(
     requests: List[Request],
     referrer: Optional[str] = None,
     use_cache: bool = False,
+    use_mql: bool = False,
 ) -> ResultSet:
-    # XXX (evanh): This function does none of the extra processing that the
-    # other functions do here. It does not add any automatic conditions, format
-    # results, nothing. Use at your own risk.
+    """
+    The main entrypoint to running queries in Snuba. This function accepts
+    Requests for either MQL or SnQL queries and runs them on the appropriate endpoint.
+    """
+
     metrics.incr("snql.sdk.api", tags={"referrer": referrer or "unknown"})
```

Comment:
Should we add a new tag to track the cutover (e.g. `lang: "snql" | "mql"`)?

Reply:
I thought about that, but I have metrics for that in the metrics layer. This metric is across all queries to Snuba, so I don't know that it would be useful to track the two languages here.

Reply:
Ok cool. I think it's probably safe to assume we won't drift and run MQL outside of the layer given the complexity required to build the indexer_mappings. So the metric in the layer should be sufficient.